exampleSite/content/about/pricing/index.md

```diff
@@ -0,0 +1,11 @@
++++
+fragment = "pricing"
+weight = 100
+# background = "light"
+
+title = "Pricing fragment"
+subtitle = """
+Pricing fragment supports **markdown** as it's subtitle.  
```

Comment:
We usual default to subtitle being just the subtitle and then have the content section used for the description. Any reason, why to do it differently here?

Reply:
I could not get `.Content` to work. Checkout the new diff. `.Content` does not render.